src/tests/Equinor.ProCoSys.IPO.MainApi.Tests/LibraryApi/FunctionalRole/LibraryApiFunctionalRoleServiceTests.cs

```diff
@@ -149,5 +149,24 @@ public async Task GetFunctionalRolesByCode_ShouldReturnCorrectProperties()
             Assert.AreEqual("ola@test.com", person.Email);
             Assert.AreEqual(new Guid("11111111-1111-2222-2222-333333333333").ToString(), person.AzureOid);
         }
+
+        [TestMethod]
+        public async Task GetFunctionalRolesByCode_ShouldReturnEncodedFunctionalRoleCode_WhenFunctionalRoleCodeIncludesAndSign()
```

Comment:
I don't know if this test is actually necessary, since it provides false security. This test would have passed even before this fix

Reply:
Hmm no the test fails without the fix!